README.md

```diff
@@ -0,0 +1,141 @@
+# talk
+
+## Requirements
+
+- MongoDB >=3.6
+- Redis >=3.2
+- NodeJS >=10
+- NPM >=6.7
+
+## Running
+
+You can install Talk using Docker or via Source. We recommend Docker, as it
+provides the easiest deployment solution going forward, as all the dependencies
+are baked and shipped with the provided
+[coralproject/talk:next](https://hub.docker.com/r/coralproject/talk) image.
+When v5 releases to master, you'll be able to select it using
+`coralproject/talk:5`.
+
+### Docker
+
+The easiest way to get started with Talk is through our published Docker image
+and provided example `docker-compose.yml` file. The following assumes that you
+have Docker and Docker Compose installed on your local machine:
+
+- Install Docker: https://docs.docker.com/install/
+- Install Docker Compose: https://docs.docker.com/compose/install/ (this is typically included in the Docker Desktop editions already)
+
+```bash
+# Create directories to persist the data in MongoDB and Redis.
+mkdir -p data/{mongo,redis}
+
+# Create the `docker-compose.yml` file to get started.
+cat > docker-compose.yml <<EOF
+version: "2"
+services:
+  talk:
+    image: coralproject/talk:next
+    restart: always
+    ports:
+      - "127.0.0.1:3000:5000"
```

Comment:
What's at port 5000?

Reply:
Only Docker defaults to 5000, as 3000 is typically considered a development port.